dracut/30coreos-installer/parse-coreos.sh

```diff
@@ -24,6 +24,110 @@ then
     echo $IGNITION_URL >> /tmp/ignition_url
 fi
 
+# Dracut networking args
+# Parse all args (other than rd.neednet) and persist those into /tmp/networking_opts
+# List from https://www.mankier.com/7/dracut.cmdline#Description-Network
+local NETWORKING_ARGS="rd.neednet=1"
+local IP_OPT=$(getarg ip=)
+local IFNAME_OPT=$(getarg ifname=)
+local ROUTE_OPT=$(getarg rd.route=)
+local BOOTDEV_OPT=$(getarg bootdev=)
+local BOOTIF_OPT=$(getarg BOOTIF=)
+local RDBOOTIF_OPT=$(getarg rd.bootif=)
+local NAMESERVER_OPT=$(getarg nameserver=)
+local PEERDNS_OPT=$(getarg rd.peerdns=)
+local BIOSDEVNAME_OPT=$(getarg biosdevname=)
+local VLAN_OPT=$(getarg vlan=)
+local BOND_OPT=$(getarg bond=)
+local TEAM_OPT=$(getarg team=)
+local BRIDGE_OPT=$(getarg bridge=)
+
+if [ ! -z "$IP_OPT" ]
```

Comment:
any opposition to running this as a for loop instead?

Reply:
Re-working it into a loop works for me